docs/BasicCommunication/OnAppRegistered/index.md

```diff
@@ -35,43 +35,44 @@ If the application resumes data successfully:
-  4. HMI must provide the user with the possibility to choose an application among a list of registered applications.
-  5. HMI must send an `OnAppActivated` notification to SDL when the user activates an app via the `UI` or `VR`.
-
+  1. Update its list of registered applications.
+  2. Store the application data sent in the `applications` parameter.
+  3. Compile and store `VRGrammars` for the `vrSynonyms` parameter, and arrange them for the user to be able to use via voice recognition. Note: The VR commands to activate an application must be accessible when viewing a different active application or the list of registered applications.
+  4. Provide the user with the possibility to choose an application among a list of registered applications.
+  5. Send an `OnAppActivated` notification to SDL when the user activates an app via the `UI` or `VR`.   
+  6. Manage application events by priority. HMI gets proirity information from _OnAppRegistered_, _UpdateAppList_, _ActivateApp_ HMI API.
 !!!
 
-!!! NOTE
-
-If a device is connected over USB and registers an application, SDL will send `OnAppRegistered` with a hash of the usb serial number as the device id.
-
-If a device is connected over Bluetooth or Wi-Fi and registers an application, SDL will send `OnAppRegistered` with a hash of the device's mac address as the device id.
+!!! NOTE   
 
+   - If a device is connected over USB and registers an application, SDL will send `OnAppRegistered` with a hash of the usb serial number as the device id.
+   - If a device is connected over Bluetooth or Wi-Fi and registers an application, SDL will send `OnAppRegistered` with a hash of the device's mac address as the device id.
+   - When the application is registered for the first time (no records in PT) PoliciesManager should not initiate prompting the User about the event.
 !!!
 
-
-
 ### Notification
 
 #### Parameters
 
 |Name|Type|Mandatory|Additional|
 |:---|:---|:--------|:---------|
-|application|[Common.HMIApplication](../../common/structs/#hmiapplication)|true||
-|ttsName|[Common.TTSChunk](../../common/structs/#ttschunk)|false|array: true<br>minsize: 1<br>maxsize: 100|
+|application|[Common.HMIApplication]|true||
+|ttsName|[Common.TTSChunk]|false|array: true<br>minsize: 1<br>maxsize: 100|
 |vrSynonyms|String|false|array: true<br>minsize: 1<br>maxsize: 100<br>maxlength: 40|
 |resumeVrGrammars|Boolean|false||
-|priority|[Common.AppPriority](../../common/enums/#apppriority)|false||
+|priority|[Common.AppPriority]|false||
+
+[Common.HMIApplication]: ../../Common/Structs/index.md#hmiapplication
+[Common.TTSChunk]: ../../Common/Structs/index.md#ttschunk
+[Common.AppPriority]: ../../Common/Enums/index.md#apppriority
```

Comment:
Directories such as `Common` and `Enums` are all lowercase, and you shouldn't include the tailing `/index.md`, that is implicit with these links. A 404 error occurs when following this links.